rsmas_logging.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+class rsmas_logger():
+
+    format = "%(levelname)s - %(message)s"
+    console_handler, file_handler = None, None
+    logfile_name = None
+    logger = logging.getLogger()
+
+    def __init__(self, console=True, file=None):
+
```

Comment:
nit: remove whitespace when it doesn't add to readability

Reply:
Which whiteface doesn't add readability: line 9 or line 11? Personally think lines above and below the method declaration do add readability.